spinn_front_end_common/interface/interface_functions/router_provenance_gatherer.py

```diff
@@ -48,6 +48,47 @@ class RouterProvenanceGatherer(object):
         # transceiver
         '_txrx',
     ]
+    __DROPPED_MC_MSG = (
```

Comment:
I highly dislike the moving of stings only used on one place to constants.

Especially when they need formatting.

Why not keep them in place with f" style formatting.

Reply:
You'd rather it was the other way? OK. Can do.